apps/dash-covid-xray/app.py

```diff
@@ -1,132 +1,74 @@
+from time import time
+
 import numpy as np
+from nilearn import image
+from skimage import draw, filters, exposure, measure
+from scipy import ndimage
+
+import plotly.graph_objects as go
+import plotly.express as px
+
 import dash
 from dash.dependencies import Input, Output, State
 import dash_bootstrap_components as dbc
 import dash_html_components as html
 import dash_core_components as dcc
-import plotly.graph_objects as go
-import plotly.express as px
-from plotly.express._imshow import _array_to_b64str
-from nilearn import image
-from skimage import draw, filters, exposure, measure
-from scipy import ndimage
-from time import time
+from dash_slicer import VolumeSlicer
 
 app = dash.Dash(__name__)
-
 server = app.server
 
+
 t1 = time()
+
 # ------------- I/O and data massaging ---------------------------------------------------
 
 img = image.load_img("assets/radiopaedia_org_covid-19-pneumonia-7_85703_0-dcm.nii")
 mat = img.affine
-img = img.get_data()
-img_1 = np.copy(np.moveaxis(img, -1, 0))[:, ::-1]
-img_2 = np.copy(np.moveaxis(img, -1, 1))
-
-l_h = img.shape[-1]
-l_lat = img.shape[0]
+img = img.get_fdata()
```

Comment:
@almarklein I'm just wondering why you used `get_fdata` here (retrieving data as floats) rather than `get_data` (as uint8)?

Reply:
I think I just copied this from the original source?

Reply:
ok I changed back to `get_data`.